Presentation.Web/Controllers/API/UserController.cs

```diff
@@ -173,5 +187,46 @@ public override HttpResponseMessage Delete(int id, int organizationId = 0)
             // NOTE: Only exists to apply optional param for org id
             return base.Delete(id, organizationId);
         }
+
+        [HttpGet]
+        [Route("api/user/rightsholders")]
+        public HttpResponseMessage GetUsersWithRightsholderAccess()
+        {
+            return _userRightsService
```

Comment:
Nu hvor attribute og convention based routes bliver blandet sammen så skal vi lige sikre os at de andre metoder stadig virker. Mener at "SendAdvis" via brugeroverblikket går igennem denne controller men vi har før set det hele falde sammen når de to route typer blandes sammen